publish.js

```diff
@@ -44,44 +44,55 @@ function find({ cwd, name, version, registry }) {
  * @param {number} code - Status code of the package (existing/new).
  * @returns {Promise<number>} Status code of the package (existing/new).
  */
-function publish({ cwd, registry, dryRun, code }) {
+function publish({ cwd, registry, tag, dryRun, code }) {
   const args = [];
 
+  tag && args.push(`--tag ${ tag }`);
   registry && args.push(`--registry ${ registry }`);
   dryRun && args.push('--dry-run');
 
-  return exec(`npm publish ${ args.join(' ') }`, { cwd })
-    .then(() => code);
+  return exec(`npm publish ${ args.join(' ') }`, { cwd }).then(() => code);
 }
 
-module.exports = function({ cwd, name, version, registry, force, dryRun, silent }) {
+module.exports = function({ cwd, name, version, registry, tag, force, dryRun, silent }) {
   const log = logger.create({ silent });
 
   return find({ cwd, name, version, registry })
     .then((code) => {
       if (!force && code === resultCode.SAME_VERSION) {
         log.info(`Package ${ bold(name) } is up-to-date, no more action required`);
-
         process.exit(resultCode.SAME_VERSION);
       }
 
       return code;
     })
-    .then((code) => publish({
-      code,
-      cwd,
-      registry,
-      dryRun
-    }))
+    .then((code) =>
+      publish({
+        code,
+        cwd,
+        registry,
+        tag,
+        dryRun
+      }))
     .then((code) => {
       if (code === resultCode.NEW_VERSION) {
-        log.info(`Package ${ bold(blue(name)) } has been updated to v${ bold(blue(version)) }`);
+        log.info(
+          `Package ${ bold(blue(name)) } has been updated to v${ bold(blue(version)) } ${ (tag &&
```

Comment:
Instead of `a && b || c` let's use `a ? b : c` as you did on `bin/cli.js#33`

Reply:
Sure will do.